client/my-sites/plugins/plugin-details.jsx

```diff
@@ -92,6 +95,10 @@ function PluginDetails( props ) {
 	const isWpcom = selectedSite && ! isJetpack;
 	const isJetpackSelfHosted = selectedSite && isJetpack && ! isAtomic;
 
+	// Header Navigation and billing period switcher.
+	const isWide = useBreakpoint( '>1280px' );
+	const [ billingPeriod, setBillingPeriod ] = useState( 'MONTHLY' );
```

Comment:
There is any chance we can have it in global state(redux)? This would allow us to:
* Keep the period selected when moving between pages.
* Easier usage on down the line components as `PluginBrowserItem`

Reply:
Yes, I had in mind. Let's do it in a follow up.